addons/sale_crm/views/sale_order_views.xml

```diff
@@ -1,60 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="sale_action_quotations_new" model="ir.actions.act_window">
-        <field name="name">Quotation</field>
-        <field name="res_model">sale.order</field>
-        <field name="view_mode">form,tree,graph</field>
-        <field name="domain">[('opportunity_id', '=', active_id)]</field>
-        <field name="context">{'search_default_opportunity_id': active_id, 'default_opportunity_id': active_id}</field>
-    </record>
-
-    <record id="sale_view_inherit123" model="ir.ui.view">
-        <field name="name">sale.order.form.inherit.sale</field>
+    <record id="view_order_form_inherit_sale_crm" model="ir.ui.view">
+        <field name="name">sale.order.form.inherit.sale.crm</field>
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//group[@name='utm_link']" position="replace">
-                <group name="utm_link" colspan="2" class="mt-0">
-                    <field name="campaign_id"/>
-                    <field name="medium_id"/>
-                    <field name="source_id"/>
-                </group>
-            </xpath>
             <xpath expr="//group[@name='technical']" position="inside">
                 <field name="opportunity_id" help="Log in the chatter from which opportunity the order originates" groups="base.group_no_one"/>
             </xpath>
         </field>
     </record>
 
-    <record id="view_quotation_tree" model="ir.ui.view">
-        <field name="name">sale.order.tree.inherit.sale.crm</field>
-        <field name="model">sale.order</field>
-        <field name="inherit_id" ref="sale.view_quotation_tree"/>
-        <field name="arch" type="xml">
-            <xpath expr="//field[@name='invoice_status']" position="after">
-                <field name="tag_ids" optional="hide" widget="many2many_tags"/>
-            </xpath>
-        </field>
-    </record>
-
-    <record id="view_order_tree" model="ir.ui.view">
-        <field name="name">sale.order.tree.inherit.sale.crm</field>
-        <field name="model">sale.order</field>
-        <field name="inherit_id" ref="sale.view_order_tree"/>
-        <field name="arch" type="xml">
-            <xpath expr="//field[@name='invoice_status']" position="after">
-                <field name="tag_ids" optional="hide" widget="many2many_tags"/>
-            </xpath>
-        </field>
-    </record>
-
-    <!-- This menu is display in CRM app when sale is installed-->
-    <menuitem
```

Comment:
leave the menuitem in this module, but with active=False and then activate the menu in both depending modules in XML
in sale_crm:
```xml
<menuitem
    id="sale_order_menu_quotations_crm"
    name="My Quotations"
    action="sale.action_quotations"
    parent="crm.crm_menu_sales"
    sequence="2"
    active='False'/> <!-- note: might not work correctly when using the menuitem syntactic sugar (not sure if this is evalled or not)
```

in sale_management_crm/sale_renting_crm:
```xml
<record id="crm.sale_order_menu_quotations_crm" model="ir.ui.menu">
    <field name="active" eval="True"/>
</record>
```

The reason I want this is that modules should not create records for other modules (which is what you're doing up to now, when creating a menuitem whose xmlid is for `sale_crm` but in a file of the `sale_management_crm` module)

Reply:
@dbo-odoo done